src/components/NavbarStudioSVGSelector.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+type Props = {
+    icon: string;
+}
+
+const NavbarStudioSVGSelector = ({ icon }: Props) => {
+    switch (icon) {
+        case 'menu':
+            return ( 
+                <svg width="20" height="14" viewBox="0 0 20 14"  xmlns="http://www.w3.org/2000/svg">
+                <path d="M1 13H19H1ZM1 7H19H1ZM1 1H19H1Z"/>
+                <path d="M1 1H19M1 13H19H1ZM1 7H19H1Z"  strokeWidth="2" strokeLinecap="round" strokeLinejoin="round"/>
+                </svg>
+            );
+        case 'close':
+            return ( 
+                <svg width="18" height="18" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg">
+                <path d="M17.6007 0.41297C17.0683 -0.119454 16.2082 -0.119454 15.6758 0.41297L9 7.07509L2.32423 0.399317C1.79181 -0.133106 0.931741 -0.133106 0.399317 0.399317C-0.133106 0.931741 -0.133106 1.79181 0.399317 2.32423L7.07509 9L0.399317 15.6758C-0.133106 16.2082 -0.133106 17.0683 0.399317 17.6007C0.931741 18.1331 1.79181 18.1331 2.32423 17.6007L9 10.9249L15.6758 17.6007C16.2082 18.1331 17.0683 18.1331 17.6007 17.6007C18.1331 17.0683 18.1331 16.2082 17.6007 15.6758L10.9249 9L17.6007 2.32423C18.1195 1.80546 18.1195 0.931741 17.6007 0.41297Z"/>
+                </svg>
+            );
+        case 'my_lesson':
+            return ( 
+                <svg width="20" height="20" viewBox="0 0 20 20"  xmlns="http://www.w3.org/2000/svg">
+                <path d="M1 4C0.45 4 0 4.45 0 5V18C0 19.1 0.9 20 2 20H15C15.55 20 16 19.55 16 19C16 18.45 15.55 18 15 18H3C2.45 18 2 17.55 2 17V5C2 4.45 1.55 4 1 4ZM18 0H6C4.9 0 4 0.9 4 2V14C4 15.1 4.9 16 6 16H18C19.1 16 20 15.1 20 14V2C20 0.9 19.1 0 18 0ZM10 12.5V3.5L15.47 7.6C15.74 7.8 15.74 8.2 15.47 8.4L10 12.5Z" />
+                </svg>
+
+            );
+        case 'management':
+            return ( 
+                <svg width="20" height="20" viewBox="0 0 20 20"  xmlns="http://www.w3.org/2000/svg">
+                <path d="M0.789971 12.78L0.699971 12.69C0.309971 12.3 0.309971 11.67 0.699971 11.28L6.78997 5.18003C7.17997 4.79003 7.80997 4.79003 8.19997 5.18003L11.49 8.47003L17.88 1.29003C18.26 0.860028 18.93 0.850029 19.33 1.25003C19.7 1.63003 19.72 2.23003 19.37 2.62003L12.2 10.69C11.82 11.12 11.16 11.14 10.75 10.73L7.49997 7.48003L2.19997 12.78C1.81997 13.17 1.17997 13.17 0.789971 12.78ZM2.19997 18.78L7.49997 13.48L10.75 16.73C11.16 17.14 11.82 17.12 12.2 16.69L19.37 8.62003C19.72 8.23003 19.7 7.63003 19.33 7.25003C18.93 6.85003 18.26 6.86003 17.88 7.29003L11.49 14.47L8.19997 11.18C7.80997 10.79 7.17997 10.79 6.78997 11.18L0.699971 17.28C0.309971 17.67 0.309971 18.3 0.699971 18.69L0.789971 18.78C1.17997 19.17 1.81997 19.17 2.19997 18.78Z" />
+                </svg>
+
+            );
+        default:
+            return <svg></svg>
+    }
```

Comment:
let's use svg like a different files and import it here using `import`-statement

https://blog.logrocket.com/how-to-use-svgs-in-react/

Reply:
Done